### DISCUSSION TITLE
Getting a Error - plugins/system/astroid/services/provider.php on line 38

### DISCUSSION THREAD

**Discussion by southpaw3229** [templaza/astroid-framework#441] [General]
I am troubleshooting another issue, and enabled Debug mode and maximum errors and on the backend, I am seeing this error at the top.  Should I be concerned?  

BACKEND ERRORS:
plugins/system/astroid/services/provider.php
on line
38

FRONT END ERRORS:
Notice: Only variables should be passed by reference in /plugins/system/astroid/services/provider.php on line 38

Warning: Array to string conversion in /libraries/astroid/framework/library/scssphp/src/Compiler.php on line 904

Warning: Array to string conversion in /libraries/astroid/framework/library/scssphp/src/Compiler.php on line 904

Running Joomla 4.40
Astroid 3.0.3
PHP 8.1

**Comment by Chacapamac** [comment]
**Yep, I see that too with the same environment settings.**
I need to put the “**Debug System**” and “**Maximum Reporting**” on.

In my Mamp server log in the “**php_error.log**” .
```
[11-Nov-2023 16:49:38 UTC] PHP Notice:  Only variables should be passed by reference in .........plugins/system/astroid/services/provider.php on line 38
[11-Nov-2023 16:49:38 UTC] PHP Warning:  Cannot modify header information - headers already sent by (output started at ........./plugins/system/astroid/services/provider.php:38) in ........./libraries/src/Input/Cookie.php on line 108
```

**For now after updates I have an error 500 and white frontend page, not sure if related????**
**Backend work fine.**

**Reply by sonvnn** [reply]
Thank you! I will check it on Monday.

**Comment by sonvnn** [comment]
I fixed this issue in https://github.com/templaza/astroid-framework/commit/9d8c76e0c0e76d56d82477dbdcafefd5f84f8ca5

**Reply by southpaw3229** [reply]
thanks for taking care of that.
